test_runner/fixtures/fast_import.py

```diff
@@ -92,8 +94,8 @@ def fast_import(
     neon_binpath: Path,
     pg_distrib_dir: Path,
 ) -> Iterator[FastImport]:
-    workdir = Path(tempfile.mkdtemp())
-    with FastImport(None, neon_binpath, pg_distrib_dir, pg_version, workdir) as fi:
+    workdir = Path(tempfile.mkdtemp(dir=test_output_dir, prefix="fast_import_"))
+    with FastImport(None, neon_binpath, pg_distrib_dir, pg_version, workdir, cleanup=False) as fi:
```

Comment:
Was `cleanup=False` for development only?

Reply:
afaiu we keep data in other tests, so I decided to add this option as well and keep it in case it fails in CI

Reply:
We keep storage data when the `--preserve-database-files` flag is set in the pytest invocation. Perhaps you could plug into that?

Reply:
indeed, added support